src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,260 @@
+        User user = new User();
+        user.setLogin("thisLoginIsFit");
+        user.setPassword("123");
+        user.setAge(21);
+        assertThrows(UserRegistrationException.class, () -> {
+            registrationService.register(user);
+        },
+                "It must be thrown UserRegistrationException for password"
+                        + user.getPassword());
+    }
+
+    @Test
+    void register_passwordMinLength_ok() {
+        User expected = new User();
+        expected.setLogin("thisLoginIsFit");
+        expected.setPassword("123456");
+        expected.setAge(21);
+        User actual = registrationService.register(expected);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_passwordIsFit_ok() {
+        User expected = new User();
+        expected.setLogin("thisLoginIsFit");
+        expected.setPassword("12345678910");
+        expected.setAge(21);
+        User actual = registrationService.register(expected);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        User user = new User();
+        user.setLogin("thisLoginIsFit");
+        user.setPassword("thisPasswordIsFit");
+        user.setAge(null);
+        assertThrows(UserRegistrationException.class, () -> {
+            registrationService.register(user);
+        },
+                "It must be thrown UserRegistrationException for null age");
+    }
+
+    @Test
+    void register_userAgeIsLessThanZero_notOk() {
+        User user = new User();
+        user.setLogin("thisLoginIsFit");
+        user.setPassword("thisPasswordIsFit");
+        user.setAge(-25);
+        assertThrows(UserRegistrationException.class, () -> {
+            registrationService.register(user);
+        },
+                "It must be thrown UserRegistrationException for negative age");
+    }
+
+    @Test
+    void register_userAgeLessThanMinAge_notOk() {
+        User user = new User();
+        user.setLogin("thisLoginIsFit");
+        user.setPassword("thisPasswordIsFit");
+        user.setAge(17);
+        assertThrows(UserRegistrationException.class, () -> {
+            registrationService.register(user);
+        },
+                "It must be thrown UserRegistrationException for age"
+                        + user.getAge());
+    }
+
+    @Test
+    void register_minAge_ok() {
+        User expected = new User();
+        expected.setLogin("thisLoginIsFit");
+        expected.setPassword("thisPasswordIsFit");
+        expected.setAge(18);
+        User actual = registrationService.register(expected);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_maxValueAge_ok() {
+        User expected = new User();
+        expected.setLogin("thisLoginIsFit");
+        expected.setPassword("thisPasswordIsFit");
+        expected.setAge(Integer.MAX_VALUE);
+        User actual = registrationService.register(expected);
+        assertEquals(expected, actual);
+    }
```

Comment:
it doesn't make sense
we don't have any logic that would handle max age (because the task doesn't require it), and it's unreasonable to process such value for age